scripts/buildjs.js

```diff
@@ -0,0 +1,26 @@
+const esbuild = require('esbuild');
+const path = require('path');
+const glob = require('glob');
+
+const inputDir = './src';
+const outputDir = './dist';
+
+// Find all '.ts' files in inputDir (recursively), excluding '.test.ts' files
+const files = glob.sync(`${inputDir}/**/*.ts`, { ignore: `${inputDir}/**/*.test.ts` });
+
+const options = {
+  format: 'cjs',
+  // Add any other options you need
```

Comment:
Do we also need to copy over `--platform=node --target=node14` to here?

Reply:
Ouhh yeah probably nice spot